docs/pages/reference/cli.mdx

```diff
@@ -1,53 +1,68 @@
 ---
 title: Teleport CLI Reference
 description: Detailed guide and reference documentation for Teleport's command line interface (CLI) tools.
-h1: Command Line (CLI) Reference
 ---
 
-Teleport is made up of three CLI tools.
+Teleport is made up of four CLI tools.
 
-- [teleport](#teleport): The daemon that runs the Auth Service, Proxy Service, and other Teleport services.
-- [tsh](#tsh): A tool that lets end users interact with Teleport.
-- [tctl](#tctl): An administrative tool that can configure the Teleport Auth Service.
+- [teleport](#teleport): A CLI tool that supports the Teleport access plane by starting and configuring various Teleport services.
+- [tsh](#tsh): A client CLI tool that allows end users to authenicate to Teleport and access resources in a cluster.
+- [tctl](#tctl): An administrative CLI tool that can configure the Teleport Auth Service.
+- [tbot](#tbot): A CLI tool that supports Machine ID, which provides short lived credentials to service accounts (e.g, a CI/CD server).
 
 (!docs/pages/includes/permission-warning.mdx!)
 
 (!docs/pages/includes/backup-warning.mdx!)
 
 ## teleport
 
-The Teleport daemon is called `teleport`. It can be configured to run one or
-more "roles" with the `--roles` flags. The arguments to `--roles` correspond to
-the following services.
-
-<Notice type="tip" scope={["cloud"]}>
-
-Teleport Cloud manages Teleport instances with the `auth` and `proxy` roles. Use
-the remaining roles to manage access to specific resources and other Teleport
-clusters.
-
+The CLI tool that supports the Teleport access plane is called `teleport`, and allows Teleport services to be managed
+over the command line:
+
+- [Auth](../architecture/authentication.mdx)
```

Comment:
I know it would repeat "Service", but the full name of each service is the "Auth Service", "Proxy Service", etc. Otherwise, I'm not sure it's clear what a user should expect each link in this list to point to.

Reply:
@rsyracuse Just checking to see if you wanted to incorporate this one